catalogs/catalog-lakehouse-iceberg/src/main/java/org/apache/gravitino/catalog/lakehouse/iceberg/converter/FromIcebergType.java

```diff
@@ -90,13 +92,15 @@ public Type primitive(org.apache.iceberg.types.Type.PrimitiveType primitive) {
       case DATE:
         return org.apache.gravitino.rel.types.Types.DateType.get();
       case TIME:
-        return org.apache.gravitino.rel.types.Types.TimeType.get();
+        return org.apache.gravitino.rel.types.Types.TimeType.of(PRECISION_MICROSECOND);
```

Comment:
Iceberg only supports one micro precision? Could you please attach the relevant document here?

Reply:
> Iceberg only supports one micro precision? Could you please attach the relevant document here?

Yes, currently only `microseconds` is supported. Here is the link: https://iceberg.apache.org/docs/nightly/schemas/